app/src/App.re

```diff
@@ -1,11 +1,26 @@
-open Css;
+module Styles = {
+  open Css;
 
-global("body", [color(Particle.Color.black()), ...Particle.Font.text]);
+  global(
+    "body",
+    [
+      color(Particle.Color.black()),
+      unsafe("-webkit-font-smoothing", "antialiased"),
+      unsafe("-moz-osx-font-smoothing:", "grayscale"),
+      ...Particle.Font.text,
+    ],
+  );
+
+  global("a", [color(Particle.Color.black()), textDecoration(none)]);
```

Comment:
Can we add `cursor: pointer` to indicate that they are clickable, trips me up even in local debug. :>